docs/get-started/best-practices/best-practices-for-harvard.md

```diff
@@ -0,0 +1,208 @@
+impact of a given compromise.
+
+## Safer Networking
+
+The goal in safer networking is to minimize the areas that an attacker can target.
+
+### Minimize Publicly Exposed Services
+
+Every port/service open to the internet will be scanned to access your servers.
+We recommend that any service/port that is not needed to be accessed by the
+public be placed behind the campus firewall. This will significantly reduce the
+number of attempts by attackers to compromise your servers.
+
+In practice this usually means that you only expose posts 80/443 which enables
+you to serve websites, while you keep all other services such as SSH,
+WordPress-logins, etc behind the campus firewall.
+
+### Strengthen SSH Logins
+
+Where possible, and if needed, logins to a Harvard service should be placed behind
+Harvardkey. For researchers however, the preferred login method is usually SSH
+and we recommend the following ways to strengthen your SSH accounts
+
+● Disable password only logins
+
+- In file `/etc/ssh/sshd_config` change `PasswordAuthentication` to `no` to disable
+tunneled clear text passwords i.e. `PasswordAuthentication no`.
+
+- Uncomment the second line, and, if needed, change yes to no.
+
+- Then run `service ssh restart`.
+
+● Use SSH keys with passwords enabled on them
+
+● If possible, enroll the SSH service with a Two-factor authentication provider
+such as DUO or YubiKey.
+
+## Attack Detection
+
+Despite the best protection, a sophisticated attacker may still find a way to
+compromise your servers and in those scenarios, we want to enhance your ability
+to detect activity that may be suspicious.
+
+### Install Crowdstrike
+
+As stated above, Crowdstrike is both an endpoint protection service and also an
+endpoint detection service. This software understands activities that might be
+benign in isolation but coupled with other actions on the device may be
+indicative of a compromise. It also enables the quickest security response.
+
+Crowdstrike can be downloaded from our repository at: agents.itsec.harvard.edu
+this software is needed for all devices owned by Harvard staff/faculty and
+available for all operating systems.
+
+### Safeguard your System Logs
+
+System logs are logs that check and track activity on your servers, including
+logins, installed applications, errors and more.
+
+Sophisticated attackers will try to delete these logs to frustrate investigations
+and prevent discovery of their attacks. To ensure that your logs are still
+accessible and available for review, we recommend that you configure your logs
+to be sent to a system separate from your servers. This can be either sending logs
+to an external file storage repository. Or configuring a separate logging system
+using **Splunk**.
+
+For help setting up logging please file a support request via our support
+ticketing system: [ServiceNow](https://harvard.service-now.com/ithelp?id=submit_ticket&sys_id=3f1dd0320a0a0b99000a53f7604a2ef9).
```

Comment:
same link question as above.

Reply:
This will be not visible on web page anyways